Bottom: 751a8c1619b9d04d9eef78abf5f0c8c0805ef72b
Top:    477738de46230b767ecd7fcacb975c530a394234
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 01:11:29 +0000

Refresh of speed-up-initial-clipping-pour

---

diff --git a/src/pour.c b/src/pour.c
index 737cffa..d1099cd 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -468,6 +468,8 @@ struct cpInfo
 static void
 subtract_accumulated (struct cpInfo *info)
 {
+  if (info->accumulate == NULL)
+    return;
   if (subtract_poly (info->accumulate, &info->pg) < 0)
     longjmp (info->env, 1);
   info->accumulate = NULL;
